### PR TITLE
fix for issue 10

### LINKED ISSUE #10: [RFE] return an error code in the case an error is found in any of the scripts
A proper implementation of error return codes needs to be developed.

### PR DESCRIPTION
**Pull request by ptrnull**
Fix to exit with an error if the number of errors was gt 0.
closes #10

### CODE REVIEW

File: info/mtu [hunk 1]
```diff
@@ -1,43 +0,0 @@
-#!/usr/bin/env bash
```
Comment (e-minguez) [inline info/mtu:1]: I think deleting the entire script because it fails is too aggressive 🗡️  Why don't we try to fix it instead? :D

**Review by e-minguez — CHANGES_REQUESTED** [review]

File: info/mtu (hunk 1 above)
Comment (ptrnull) [reply info/mtu]: ups! I did it again, move it back to it's place, thanks for the heads up :)

File: utils [hunk 2]
```diff
@@ -39,10 +39,14 @@ msg() {
   echo >&2 -e "${1-}"
```
Comment (e-minguez) [inline utils:39]: I guess we can then output the rest of the messages to stdout
```suggestion
  echo -e "${1-}"
```

File: utils [hunk 3]
```diff
@@ -39,10 +39,14 @@ msg() {
...
 }
 
+error() {
+  echo 1>&2 -e "${1-}"
```
Comment (e-minguez) [inline utils:43]: And keep this only to stderr
```suggestion
  echo >&2 -e "${1-}"
```

**Review by e-minguez — COMMENTED** [review]
Thanks @ptrnull I've added a few comments. PTAL /cc @dcritch

**Pull request merged**